Documentation edits to the Getting Started Section (#1510)
eclipse-cyclonedds/cyclonedds: docs/manual/getting_started/helloworld/helloworld_key_steps.rst

   participation within a DDS domain by creating a **Domain Participant
   entity**.
#. Create a **Data topic** with the data type described in a data
   model. The data types define the structure of the Topic. The Topic is
   therefore, an association between the topic's name and a datatype.
   QoSs can be optionally added to this association. The concept Topic
   therefore discriminates and categorizes the data in logical classes
   and streams.
#. Create the **Data Readers** and **Writers** entities 
   specific to the topic. Applications may want to change the default
   QoSs. In the ``Hello world!`` Example, the ``ReliabilityQoS`` is changed
   from its default value (``Best-effort``) to ``Reliable``.
#. When the previous DDS computational entities are in place, the
   application logic can start writing or reading the data.

At the application level, readers and writers do not need to be aware of
each other. The reading application, now called Subscriber, polls the
data reader periodically until a publishing application, now called
The publisher writes the required data into the shared topic, namely
``HelloWorldData_Msg``.

> The data type is described using the OMG IDL.
> Language <http://www.omg.org/gettingstarted/omg_idl.htm>`__ located in

Reviewer: Minor fix (the old URL also doesn't point anywhere). The actual new link here should probably get put into the external link file.
```suggestion
The data type is described using `OMG IDL <https://www.omg.org/spec/IDL/4.2>`__
within the
```

Author: Comments incorporated
